shared/mysql/compose.yml

```
@@ -0,0 +1,23 @@
+services:
+  mysql1:
+    image: mysql:8.0
+    container_name: mysql1
```

I propose to stick to the convention of putting `benchi-` in front of the container name (prevents conflicts, in case you have a `mysql` container already running, also easier to find if something is left over after a non-graceful shutdown).
```suggestion
  benchi-mysql:
    image: mysql:8.0
    container_name: benchi-mysql
```